File: translation_output/misleading_test/magicoder/codenet/python/java/p02835.java
<｜begin▁of▁sentence｜>You are an exceptionally intelligent coding assistant that consistently delivers accurate and reliable responses to user instructions.

@@ Instruction
Translate the following python code to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
5 7 9

Expected output:
wi

```
a,b,c = map(int, input().split())
print('win' if a+b+c<22 else 'bust')
```

@@ Response
Here is the Java version of the given Python code:

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] input = scanner.nextLine().split(" ");
        int a = Integer.parseInt(input[0]);
        int b = Integer.parseInt(input[1]);
        int c = Integer.parseInt(input[2]);

        if (a + b + c < 22) {
            System.out.println("win");
        } else {
            System.out.println("bust");
        }
    }
}
```

This Java code reads the input from the user, splits it into three integers, and then checks if the sum of these integers is less than 22. If it is, it prints "win", otherwise it prints "bust".<｜end▁of▁sentence｜>

File: translation_output/misleading_test/magicoder/codenet/java/python/p02835.py
<｜begin▁of▁sentence｜>You are an exceptionally intelligent coding assistant that consistently delivers accurate and reliable responses to user instructions.

@@ Instruction
Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
5 7 9

Expected output:
vin

```
import java.util.Scanner;

public class p02835{

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int a = sc.nextInt();
		int b = sc.nextInt();
		int c = sc.nextInt();
		if(a+b+c >= 22) {
			System.out.println("bust");
		}else {
			System.out.println("win");

		}
	}

}

```

@@ Response
```python
# Python code

# Get input from user
a, b, c = map(int, input().split())

# Check if the sum of the three numbers is greater than or equal to 22
if a + b + c >= 22:
    print("bust")
else:
    print("win")
```

This Python code does the same thing as the Java code. It reads three integers from the user, adds them together, and then checks if the sum is greater than or equal to 22. If it is, it prints "bust". If it's not, it prints "win".<｜end▁of▁sentence｜>
